.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,53 +1,28 @@
 ## Description
 
-<!--- 
-Describe the purpose of this PR along with any background information and the impacts of the proposed change. 
-For the benefit of the community, please do not assume prior context.
-
-Provide details that support your chosen implementation, including: 
-- breaking changes
-- alternatives considered
-- changes to the API
-- demos (screenshots, videos) if you find that useful
-- etc.
--->
-
-
-## References
-
-<!--- 
-Include any links supporting this change such as a:
-
-- GitHub Issue/PR number addressed or fixed
-- Auth0 Community post
-- StackOverflow post
-- Support forum thread
-- Related pull requests/issues from other repos
-
-If there are no references, simply delete this section.
--->
```

Comment:
IMO, references are redundant to description; it should be an all encompassing excerpt that describes the set of changes.